build/gen-type.js

```diff
@@ -5,6 +5,12 @@ const { noElPrefixFile } = require('./common')
 
 const outsideImport = /import .* from '..\/(.*?)\/src\/.*/
 
+// global.d.ts
+const globalPath = path.resolve(__dirname, '../typings/vue-shim.d.ts')
+fs.readFile(globalPath, (err, data) => {
```

Comment:
Why not just use `fs.copyFileSync` instead ? And I wonder if it is appropriate to copy the whole `vue-shim.d.ts`, cause the global declarations may cause pollution.